var/spack/repos/builtin/packages/cddlib/package.py

```diff
@@ -13,16 +13,16 @@ class Cddlib(AutotoolsPackage):
     and extreme rays of a general convex polyhedron in R^d given by a system
     of linear inequalities"""
 
-    homepage = "https://www.inf.ethz.ch/personal/fukudak/cdd_home/"
-    url      = "ftp://ftp.math.ethz.ch/users/fukudak/cdd/cddlib-094h.tar.gz"
-
-    version('0.94h', sha256='fe6d04d494683cd451be5f6fe785e147f24e8ce3ef7387f048e739ceb4565ab5')
+    homepage = "https://people.inf.ethz.ch/fukudak/cdd_home"
+    url      = "https://github.com/cddlib/cddlib/archive/refs/tags/0.94h.tar.gz"
+    maintainers = ['NessieCanCode']
+    version('0.94h', sha256='7382782c3834214b022c8b2898ed775a7bf915f2cb2acb73fa045d6fd9a3de33')
```

Comment:
Newer versions are available. Since you're updating the hash anyway you could also add the newest version.

Reply:
it was a greed move on my part, I only needed 0.94h, as the ftp server that was listed was timing out. but you are right, I can, so I've since added several of the version releases all the way to the latest version.